### DISCUSSION TITLE
Manage multiple clusters with one argocd

### DISCUSSION THREAD

**Discussion by Meza1923** [argoproj/argo-cd#8730] [Q&A] [answered]
Hey, 

I'm trying to connect argocd to remote cluster, but I get error.
I run the command:
```
argocd cluster add https://<api-url>:6443 
```
and the error
```
FATA[0000] Context https://<api-url>:6443 does not exist in kubeconfig
```
Before I execute this commad I have loged to the cluster from the same server.
Can you help me to solve this issue? there is other way? maybe from the ui or using crd?

Thanks a lot,
Guy.

**Comment by todaywasawesome** [comment, marked as answer]
It's looking for a context name like `argocd cluster add docker-desktop` that will pull from your Kubeconfig the proper url and config. https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_cluster_add/

**Reply by Meza1923** [reply]
thank you, I will try that.

**Comment by Meza1923** [comment]
The solution of @todaywasawesome worked.
I read the, kube / config file, and there I saw:
current-context: default / <name-of-the-cluster>: 6443

so I ran,
`` `
argocd cluster add default / <name-of-the-cluster>: 6443
`` `

There is other way than use CONTEXT?

**Reply by AmitKatyal-Sophos** [reply]
I am getting the below error while adding the cluster.
➜  argocd-ecr-creds git:(ztnaaas-helm-test) ✗ argocd cluster add arn:aws:eks:us-west-2:XXXXX:cluster/ztnaaas-XXXXX --system-namespace argocd
WARNING: This will create a service account `argocd-manager` on the cluster referenced by context `arn:aws:eks:us-west-2:XXXXX:cluster/ztnaaas-XXXXXX` with full cluster level admin privileges. Do you want to continue [y/N]? y
FATA[0009] Failed to create service account "argocd-manager" in namespace "argocd": Unauthorized
